docs/software-engineering-policies/SourceControl/SourceControlPolicy.md

```diff
@@ -1,3 +1,48 @@
 # Source Control
 
-(Under construction)
\ No newline at end of file
+## Repositories
+
+- All code **must** be stored in a source code repository.
+- Git is the prefered method of source control
+- If hosted at the UKHO, these must be used on the central Azure DevOps Server system.
+- It is acceptable to use our official Azure DevOps instance on Azure to provide version control.
+- It is aceptable (and preferred) to use GitHub under our official UKHO organisation.
+- **No other source control systems are permitted.**
+- Any source code found in any other source control method, must first be migrated and proven to build via a CI pipeline
+- All new source will host a primary branch called _trunk_ or _main_ over _master_, every effort of existing source code should be made to move to this setup. 
```

Comment:
"every effort of existing source code should be made to move to this setup. " - there is a separate policy coming up about inclusive language which covers it.  Can you please remove this as it contains a slightly different message?

Reply:
```suggestion
- All new source will host a primary branch called _trunk_ or _main_ over _master_.
```